src/CenterEdge.Async/AsyncHelper.cs

```diff
@@ -51,10 +51,46 @@ public static void RunSync(Func<Task> task)
             }
         }
 
+        /// <summary>
+        /// Executes an async <see cref="Task"/> method with no return value synchronously.
+        /// </summary>
+        /// <param name="task"><see cref="Task"/> method to execute.</param>
+        /// <param name="state">State to pass to the method.</param>
+        /// <remarks>
+        /// DO NOT use this methods unless absolutely necessary. Calling async code from sync code is an anti-pattern
```

Comment:
```suggestion
        /// DO NOT use this method unless absolutely necessary. Calling async code from sync code is an anti-pattern
```

I see this note all over the place tho, no hill for me